fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/EnforceAndCostTask.java

```diff
@@ -356,6 +357,12 @@ private boolean canGenerateOneStageAgg(GroupExpression childBestExpr) {
         }
 
         PhysicalHashAggregateOperator aggregate = (PhysicalHashAggregateOperator) groupExpression.getOp();
+        // Must do one stage aggregate If there are exchange_bytes/_speed()
+        if (aggregate.getAggregations().values().stream().anyMatch(aggFunc ->
```

Comment:
why must do one stage aggregate?

Reply:
if using two stage, the exchange data becomes smaller, which is not helpful to perf exchange.